examples/clear_color.rs

```diff
@@ -0,0 +1,21 @@
+use conifer::prelude::*;
+fn main() -> Result<(), Box<dyn Error>> {
+    let img_pine = load_image("examples/images/pine.png")?;
+    let mut c = Config::auto()?;
+    let green = Canvas::from_color(c.screen_width(), c.screen_height(), 0, 100, 0);
+    c.run(move |canvas, swipe, _delta_time| {
+        if let Some(s) = swipe {
+            if s.finished {
+                return Ok(RunResponse::Exit);
+            }
+        }
+        canvas.copy_from_canvas(&green);
```

Comment:
This gives me an error, canvas.copy_from_canvas(&green) does not make a vec of the right size.

Reply:
hmm, i'm wondering if our framebuffers are different somehow, for me this works, but i'm wondering if you have a different bytes per pixel

Reply:
height:480, width:800, line length:7680, bpp:4. It detects the line length of my HDMI monitor, even though it detected the size of the touchscreen.

Reply:
whoa.. interesting ... i'm wondering if we can ditch the concept of linelength, i'm not even sure what we use it for that can't be done with width

![image](https://user-images.githubusercontent.com/294042/91669001-4af1b480-eac6-11ea-95f0-75f07b5d0193.png)

Reply:
Had to unplug my HDMI monitor for it to work. So far the line length is just width*bpp, I don't know whether it can be otherwise.